pitop/processing/algorithms/ball_detect.py

```diff
@@ -36,40 +36,60 @@ def __init__(self, color):
         }
         self.match_limit = match_limits[color]
 
-        self.center_points = deque(maxlen=DETECTION_POINTS_BUFFER_LENGTH)
-        self.radius = 0
-        self.angle_from_center = None
+        self._center_points_cv = deque(maxlen=DETECTION_POINTS_BUFFER_LENGTH)
```

Comment:
Why is it called `center_points_cv`? They are co-ordinates and have nothing to do with OpenCV?

Reply:
Our coordinate system for the user is referenced from the centre of the frame, so this is here to explicitly state these are OpenCV coordinates referenced from the top left of the frame

Reply:
Can you explain a little more about what you mean here? Perhaps a user code example?
My issue is more that it's not "opencv coordinates", top-left is standard for `(0,0)` in the SDK and elsewhere, so I would argue that this should just be `center_points` and wherever the user is getting 'centered/calibrated' co-ordinates should be made clear wherever they are used.